undertow/src/main/java/org/wildfly/extension/undertow/filters/ModClusterDefinition.java

```diff
@@ -67,110 +67,110 @@
 
     public static final AttributeDefinition MANAGEMENT_SOCKET_BINDING = new SimpleAttributeDefinitionBuilder(Constants.MANAGEMENT_SOCKET_BINDING, ModelType.STRING)
             .setAllowExpression(true)
-            .setAllowNull(false)
+            .setRequired(true)
             .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.SOCKET_BINDING_REF)
             .setRestartAllServices()
             .build();
 
     public static final AttributeDefinition ADVERTISE_SOCKET_BINDING = new SimpleAttributeDefinitionBuilder(Constants.ADVERTISE_SOCKET_BINDING, ModelType.STRING)
             .setAllowExpression(true)
-            .setAllowNull(false)
+            .setRequired(false)
```

Comment:
Doesn't this require a transfomer to reject the attribute if it is undefined?
Current version will accept an undefined ADVERTISE_SOCKET_BINDING but this will fail for any legacy version.